### PR TITLE
Added Example with turn_on_action:

### PR DESCRIPTION
**Pull request by lorenzofattori**
It took me 2 hours to make it work. It deserves an example

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.philips_js.markdown [hunk 1]
```diff
@@ -35,3 +35,15 @@ Configuration variables:
...
 </p>
+
+
+```yaml
+# Example configuration.yaml with turn_on_action
+media_player:
+  - platform: philips_js
+    host: 192.168.1.99
+    turn_on_action: 
+    service: wake_on_lan.send_magic_packet
```
Comment (frenck) [inline source/_components/media_player.philips_js.markdown]: Is this indenting correct?

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @lorenzofattori, thank you so much for sending in this PR!

I've reviewed it and left a question. Could you please take a look? 👍

**Comment by lorenzofattori** [comment]
Hi @frenck , sorry my first home assistant file and first time on github..
Fixed identation, now is correct.

File: source/_components/media_player.philips_js.markdown (hunk 1 above)
Comment (lorenzofattori) [reply source/_components/media_player.philips_js.markdown]: Hi @frenck , sorry my first home assistant file and first time on github..
Fixed identation, now is correct.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thank you for adding this @lorenzofattori! 
That is really helpful!

PS: Congrats on getting your first documentation PR merged. Got some more? 😉 

bitmoji

**Pull request merged**